app/components/List.jsx

```diff
@@ -86,42 +87,35 @@ const styles = theme => ({
   }
 });
 
+const lastModifiedFormatter = new Intl.DateTimeFormat()
+
 class ListAll extends Component {
 
   constructor(props) {
     super(props);
     const { classes } = props;
     this.classes = classes;
-    this.state = { resumes: [], filteredResumes: [] };
-
-    this.filterList = this.filterList.bind(this);
-  }
+    this.state = { resumes: [], abortFetch: () => {} };
 
-  componentDidMount() {
-    authorizedFetch(`/resumes`)
-      .then(res => res.json())
-      .then(data => {
-        this.setState({ resumes: data, filteredResumes: data });
-      });
+    this.searchResumesWithDebounce = debounce(this.searchResumes.bind(this), 200);
```

Comment:
We should probably clean this up and abort pending fetch in a `componentWillUnMount` ?

Reply:
You're right!

Reply:
I guess that to mirror this, the debounce should be created on `componentDidMount`?

Reply:
Yes probably.

It's a perfect use-case for useEfect but I imagine that the version of react is too old ?

Reply:
Correct. :)